cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/WhereTest.scala

```diff
@@ -467,39 +470,78 @@ class WhereTest extends DocumentingTest {
-          }
-          p("The same rule applies to pattern comprehensions:")
-          query("""MATCH (a:Person {name: 'Andy'})
-                  #RETURN [(a)-->(b WHERE b:Person) | b.name] AS friends""".stripMargin('#'),
-            ResultAssertions { r =>
-              val rows = r.toList
-              rows.length shouldBe 1
+    }
+    section("Pattern element predicates", "pattern-element-predicates") {
+      p("`WHERE` clauses can be added to <<cypher-patterns,pattern elements>> in order to specify additional constraints.")
+      section("Node pattern predicates", "node-pattern-predicates") {
+        p("`WHERE` can appear inside a node pattern in a `MATCH` clause or a pattern comprehension:")
+        query("""WITH 30 AS minAge
+                #MATCH (a:Person WHERE a.name = 'Andy')-[:KNOWS]->(b:Person WHERE b.age > minAge)
+                #RETURN b.name""".stripMargin('#'),
+          ResultAssertions { r =>
+            r.toList shouldBe List(Map("b.name" -> "Peter"))
+          }) {
+          resultTable()
+        }
+        p("The same rule applies to pattern comprehensions:")
+        query("""MATCH (a:Person {name: 'Andy'})
+                #RETURN [(a)-->(b WHERE b:Person) | b.name] AS friends""".stripMargin('#'),
+          ResultAssertions { r =>
+            val rows = r.toList
+            rows.length shouldBe 1
 
-              inside(rows.head("friends")) {
-                case friends: Seq[_] =>
-                  friends should contain theSameElementsAs Seq("Peter", "Timothy")
-              }
-            }) {
-            resultTable()
-          }
+            inside(rows.head("friends")) {
+              case friends: Seq[_] =>
+                friends should contain theSameElementsAs Seq("Peter", "Timothy")
+            }
+          }) {
+          resultTable()
+        }
+      }
+      section("Relationship pattern predicates", "relationship-pattern-predicates") {
+        p("`WHERE` can also appear inside a relationship pattern in a `MATCH` clause:")
+        query("""WITH 2000 AS minYear
+                #MATCH (a:Person)-[r:KNOWS WHERE r.since < minYear]->(b:Person)
+                #RETURN r.since""".stripMargin('#'),
+          ResultAssertions { r =>
+            r.toList shouldBe List(Map("r.since" -> 1999))
+          }) {
+          resultTable()
+        }
+        p("However, it cannot be used inside of variable length relationships:")
+        query(
+          """WITH 2000 AS minYear
+            #MATCH (a:Person)-[r:KNOWS*1..3 WHERE r.since > b.yearOfBirth]->(b:Person)
+            #RETURN r.since""".stripMargin('#'),
+          ErrorAssertions(_.getMessage should startWith("Relationship pattern predicates are not supported for variable-length relationships"))
+        )(
+          p("this would lead to an error.")
+        )
```

Comment:
```suggestion
        p("However, it cannot be used inside of variable length relationships, as this would lead to an error. For example:")
        query(
          """WITH 2000 AS minYear
            #MATCH (a:Person)-[r:KNOWS*1..3 WHERE r.since > b.yearOfBirth]->(b:Person)
            #RETURN r.since""".stripMargin('#'),
          ErrorAssertions(_.getMessage should startWith("Relationship pattern predicates are not supported for variable-length relationships"))
        )
```

Reply:
This change removes the hanging sentence after the example.